Library/views.py

```diff
@@ -93,15 +100,17 @@ def post(self):
             login_user(user)
             if request.args.get("next"):
                 return redirect(request.args.get("next"))
-            return redirect(url_for("main.dashboard"))
+            return redirect(url_for("main.admin_dashboard"))
         flash("Invalid Credentials!")
         return redirect(url_for("main.admin"))
 
 
 @main.route("/dashboard", methods=["GET"])
 @login_required
 def dashboard():
-    copies = Copy.query.filter_by(issued_by=current_user.id).all()
+    copies = db.session.query(Book.id, Book.name, Book.author, Copy.date_issued, Copy.date_return) \

```

Comment:
What does this changes ?

Reply:
The issue book page don't show the Author name and Book name, so we need to join both the tables from Book and Copy, where we join the two tables by book_id and search by user id